src/jsdoccomment.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable jsdoc/imports-as-dependencies -- https://github.com/gajus/eslint-plugin-jsdoc/issues/1114 */
```

Comment:
Isn't this linting directive still needed to pass linting?

Reply:
It passes without it.